org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/xpect/ConstructorAndConsts.wlk.xt

```diff
@@ -34,11 +34,20 @@ class GolondrinaUruguaya inherits Golondrina {
 		esBotija = false
 	}
 	
-	// XPECT errors --> "Variable esBotija is never assigned in this constructor" at "constructor(_energia, _esBotija) = self(_energia)"
-	constructor(_energia, _esBotija) = self(_energia)
+	// XPECT errors --> "Variable esBotija is never assigned in this constructor" at "constructor(_energia, _esBotija) = super(_energia)"
```

Comment:
Maybe we would need another test, similar to this but for a class without explicit constructors. E.g.
class C { const x; constructor(_x) { x = _x }
class D inherits C { const y } ==> invalid!

Reply:
Added, but it is a bit annoying the way XPECT forces us to both have an ERROR and a WARNING (Unused variable):

```javascript
class C { 
	const x
	constructor(_x) {
		x = _x
	} 
}

class D inherits C {
	/* XPECT errors ---
		"Unused variable" at "y"
		"Variable is never assigned" at "y"
	--- 
		XPECT warnings ---
		"Unused variable" at "y"
		"Variable is never assigned" at "y"
	---
	*/
	const y
}
```

Reply:
Strange. Maybe we should fine-tune the warnings to avoid notifying them in case there is a more severe error in the same line or so. But not now, of course.